service/deployer/installer/installer.go

```diff
@@ -7,6 +7,8 @@ import (
 	micrologger "github.com/giantswarm/microkit/logger"
 
 	"github.com/giantswarm/draughtsman/flag"
+	"github.com/giantswarm/draughtsman/service/deployer/installer/configurer"
+	configurerspec "github.com/giantswarm/draughtsman/service/deployer/installer/configurer/spec"
```

Comment:
This path for me suggests that there is an "spec implementation" of the configurer. Why not having the spec in `github.com/giantswarm/draughtsman/service/deployer/installer/configurer`?